Lib/json/tool.py

```diff
@@ -17,27 +17,27 @@
 _color_pattern = re.compile(r'''
     (?P<key>"(\\.|[^"\\])*")(?=:)           |
     (?P<string>"(\\.|[^"\\])*")             |
+    (?P<number>NaN|-?Infinity|[0-9\-+.Ee]+) |
```

Comment:
Isn't this a unrelated change?

Reply:
Yes and no. An earlier version of json.tool's coloring did have this line to support number coloring as well. It got removed due to bikeshedding on the original PR. I felt like bringing it back here because with themes you can choose what color a number will have. If the regex wouldn't recognize it, you wouldn't be able to choose.